backend/btrixcloud/pages.py

```diff
@@ -127,6 +176,10 @@ def _get_page_from_dict(
             loadState=page_dict.get("loadState"),
             status=status,
             mime=page_dict.get("mime", "text/html"),
+            filename=page_dict.get("filename"),
+            depth=page_dict.get("depth"),
+            seed=page_dict.get("seed"),
```

Comment:
```suggestion
            seed=page_dict.get("seed") or False,
```

Looks like this is either true or null, maybe should seed to false to be consistent.
Also, perhaps better to rename to isSeed on the pages model to be clearer? (and in the crawler later)

Reply:
Done